batch/batch/front_end/front_end.py

```diff
@@ -1419,26 +1701,29 @@ async def create_update(request, userdata):
     except ValidationError as e:
         raise web.HTTPBadRequest(reason=e.reason)
 
-    update_id, _ = await _create_batch_update(batch_id, update_spec['token'], update_spec['n_jobs'], user, db)
+    n_jobs = update_spec['n_jobs']
+    n_job_groups = update_spec.get('n_job_groups', 0)
+
+    update_id, _, _ = await _create_batch_update(batch_id, update_spec['token'], n_jobs, n_job_groups, user, db)
```

Comment:
Hmm. Ignoring this information seems odd to me. It seems strictly better to include this information in the HTTP response. What was the reasoning for not including it?

Reply:
The aioclient didn't need that information so it was not included.

```
    async def _create_update(self) -> int:
        self._raise_if_not_created()
        update_spec = self._update_spec()
        update_json = await (
            await self._client._post(f'/api/v1alpha/batches/{self.id}/updates/create', json=update_spec)
        ).json()
        return int(update_json['update_id'])

    async def _commit_update(self, update_id: int) -> Tuple[int, int]:
        self._raise_if_not_created()
        commit_json = await (
            await self._client._patch(f'/api/v1alpha/batches/{self.id}/updates/{update_id}/commit')
        ).json()
        return (int(commit_json['start_job_group_id']), int(commit_json['start_job_id']))
```

I'll rewrite the client to always use that information rather than assuming the ids start with 1 in creation.